transport/src/main/java/io/netty/channel/ChannelOutboundBuffer.java

```diff
@@ -829,4 +810,25 @@ Entry recycleAndGetNext() {
             return next;
         }
     }
+
+    private static final class ChannelWritabilityChangedTask implements Runnable {
+        private final Channel channel;
+        private boolean writable = true;
+
+        ChannelWritabilityChangedTask(Channel channel) {
+            this.channel = channel;
+        }
+
+        @Override
+        public void run() {
+            if (channel.isActive()) {
+                boolean newWritable = channel.isWritable();
+
+                if (writable != newWritable) {
```

Comment:
@trustin yeah it's ok ti fire more often but should try to make it at least as "correct" as possible. So you agree this change is ok ?